### PR TITLE
Added telugu translation

### PR DESCRIPTION
**Pull request by devulapallisai**

### CODE REVIEW

File: src/_locales/te/messages.json [hunk 1]
```diff
@@ -0,0 +1,477 @@
...
+  "translatorCredit": {
+    "message": "This add-on has been translated into English by $TRANSLATORS$.",
+    "description": "The credit text for the translator. See https://github.com/TinyWebEx/common/blob/master/CONTRIBUTING.md#translator-credit-inside-of-add-on for how to translate this.",
+    "placeholders": {
+      "translators": { "content": "$1", "example": "<a href=\"https://github.com/rugk/\">@rugk</a>" }
+    },
+    "hash": "a6f5deecd624d5f8cd58350c72dd8fdf"
+  },
+  "translatorLink": {
+    "message": "https://github.com/rugk",
```
Comment (rugk) [inline src/_locales/te/messages.json:417]: You can credit yourself here and provide a link (in `translatorLink`) and a name (in `translatorUsername`). Also, you can add yourself to the `CONTRIBUTORS.md` file and the changelog as a translator to [give credits here](https://github.com/TinyWebEx/common/blob/master/CONTRIBUTING.md#credit).

See https://github.com/TinyWebEx/common/blob/master/CONTRIBUTING.md#translations for how these placeholders are translated, especially [the official doc](https://developer.mozilla.org/docs/Mozilla/Add-ons/WebExtensions/Internationalization#Providing_localized_strings_in__locales).

**Review by rugk — CHANGES_REQUESTED** [review]
Hi @dvergeylen,
thanks for your first contribution to this project! :tada: :+1: 🏅
I hope you'll like this project and enjoy hacking on it… :smiley:

To honor your work, you should be credited in the extension. To do that, you need to provide your name and a link in the extension.

Also notice there are more things to translate like the add-on listings on AMO (addons.mozilla.org) etc. [See the contributing guide for more information.](https://github.com/TinyWebEx/common/blob/master/CONTRIBUTING.md#translation-of-add-on-listings)

Please subscribe to #138 to get notified for new updates on translations etc. :smiley:

File: src/_locales/te/messages.json (hunk 1 above)
Comment (dvergeylen) [reply src/_locales/te/messages.json:417]: I think you meant @devulapallisai 🙃

**Comment by devulapallisai** [comment]
Changed requested changes. Check it. If there are any problems I would correct it.

**Review by rugk — APPROVED** [review]
Thanks a lot for the fast response and translating all this! I'll merge it and add a note to the Changelog!

**Comment by rugk** [comment]
BTW: Next time, try to avoid doing a pull request from the master branch, because you can run into problems when you have a "non-clean" master that does not follow this repo here (i.e. "upstream"). [See this article for details](https://help.github.com/en/articles/proposing-changes-to-your-work-with-pull-requests). Anyway, this is only a tip for the next time. :smiley:

**Review by rugk — APPROVED** [review]

**Pull request merged**